### DISCUSSION TITLE
Best practice on pooling the connections

### DISCUSSION THREAD

**Discussion by diorahman** [taocpp/taopq#3] [Q&A] [answered]
Hi,

Do you have best practices or snippets on how we tackle connections pooling?

Thanks!

**Comment by ColinH** [comment, marked as answer]
Hi! Have you already taken a look at `connection_pool.hpp`? It's not (yet) documented, but feel free to ask any question here. And if it doesn't fit your use case, we'd be interested to hear more about it.
